src/Sylius/Bundle/WebBundle/Resources/views/Backend/Customer/_form.html.twig

```diff
@@ -8,30 +8,44 @@
             {{ form_row(form.lastName) }}
             {{ form_row(form.birthday) }}
             {{ form_row(form.gender) }}
-            {{ form_row(form.user.plainPassword) }}
-            {{ form_row(form.user.enabled) }}
             {{ form_row(form.groups) }}
+            {% if form.user is defined %}
+                {% if form.user.vars.data.id is not defined %}
```

Comment:
form.user.id would look for ``id`` field on ``user`` child of  the main form. Here @lchrusciel wants to get the data.